src/components/Input/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import "./Input.css";
+
+interface InputProps {
+  id: number;
```

Comment:
What is the purpose of the `id` property here? What is its function?

Reply:
Removed ID from prop